### ISSUE TITLE
FEServer "True Hit" setting

### ISSUE THREAD

**Issue by ghost** [eliatlarge/FEMultiPlayer-V2#109]
Issue type: `feature request`
#### Description:

So despite the project's main github page advertising FEMP as being free from the clutches of True Hit, somehow, someway, it made it in anyways according to @eliatlarge . The community on both the sub and Discord have been very vocal on and divided about True Hit - however, at least Pwntagonist and a few others were in agreement that it should be a setting via FEServer.

@eliatlarge I'm pretty sure you've [seen my setting thread](https://www.reddit.com/r/FEMP/comments/4nokoc/rule_settings_via_feserver/), yes? Perhaps I should rename this so that all the settings from that thread are implemented (_would they be difficult since they mainly just adjust formulas and impose limits?_) but I figured at least for now, an option to enable / disable True Hit should be present. I also think True Hit should be off by default because that's what our original project description mentions as a "feature deliberately removed" - however what the default is, is up to you.

"True Hit" also explains why everyone misses with the Vegas Modifier.

**Comment by eliatlarge** [comment]
I wonder if including so many settings would clutter the file. It's too bad that Java's `properties` doesn't seem to let us auto organize when the config generates, as far as I can tell.

**Comment by SapphireTactician** [comment]
Well, this should have a separate file from the one used by the client, which makes it a little less messy.

I will admit managing these settings via a text editor may be a bit unwieldy for some, but that's just the nature of the beast and why this should be implemented as GUI expansion for FEServer, with the option to export / import a config file for faster set-up.

**Comment by wellme** [comment]
This has been implemented in one of my branch (true-hit-toggle), waiting on #136 to be merged before creating another PR (I don't into github and I'm afraid it would break something because I've misplaced some commits).

**Comment by wellme** [comment]
Implemented with PR #140
